packages/features/eventtypes/lib/types.ts

```diff
@@ -159,3 +159,42 @@ export type TabMap = {
   workflows?: React.ReactNode;
   payments?: React.ReactNode;
 };
+
+export type SettingsToggleClassNames = {
+  switchContainer?: string;
+  label?: string;
+  description?: string;
+  children?: string;
+};
+
+export type SwitchClassNames = {
+  container?: string;
+  thumb?: string;
+};
+
+export type InputClassNames = {
+  container?: string;
+  label?: string;
+  input?: string;
+  addOn?: string;
```

Comment:
What does `addOn` refer to ?

Reply:
![Screenshot 2024-11-12 at 2 31 37 PM](https://github.com/user-attachments/assets/754c78b1-ec4a-48de-835b-6e05b627d900)
the minutes thingy in the end
basically the addOnPrefix and addOnSuffix on the input fields

Reply:
Gotcha, based on your work on classes do you think that instead it could be called `unit` as in it denotes what the input field value is. Personally, `addOn` is not clear to me. What do you think?

Reply:
The `addOn` is a common class name for the input field. While it currently represents a unit, this is not consistent across all cases.